cg/services/orders/submitters/generic_order_submitter.py

```diff
@@ -0,0 +1,34 @@
+"""Module for a generic order submitter."""
+
+from cg.models.orders.order import OrderIn
+from cg.services.orders.store_order_services.store_generic_order import StoreGenericOrderService
+from cg.services.orders.submitters.order_submitter import OrderSubmitter
+from cg.services.orders.validate_order_services.validate_generic_order import (
+    ValidateGenericOrderService,
+)
+
+
+class GenericOrderSubmitter(OrderSubmitter):
+    """
+    Class for submitting generic orders.
+    This class is used to submit orders for the following workflows:
+    - Balsamic
+    - Balsamic QC
+    - Balsamic UMI
+    - MIP DNA
+    - MIP RNA
+    - Tomte
+    """
+
+    def __init__(
+        self,
+        order_validation_service: ValidateGenericOrderService,
+        order_store_service: StoreGenericOrderService,
+    ):
+        self.order_validation_service = order_validation_service
+        self.order_store_service = order_store_service
+
+    def submit_order(self, order_in: OrderIn) -> dict:
+        """Submit a generic order."""
```

Comment:
What is submitting exactly? validating, storing and returning a dict?

Reply:
Ideally submitting an order later on would be:
1. Consuming a json order / submitted order via the front end
2. Validating the order using a validator service
3. Translating the order into relevant database models via a transfer service
4. Storing these models in the database by ideally a single store service
5. Returning a Response to the end point that can be evaluated there.

This way we can encapsulate all concerns into their own modules with a single responsibility. Right now it is a bit away from that still